app/views/admin/historical_accounts/legacy_index.html.erb

```diff
@@ -10,7 +10,7 @@
     <%= tab_navigation_for(@person) %>
 
     <% if @historical_accounts.present? %>
-    <table id="person-topical-events" class="table table-bordered table-striped">
+    <table id="person-historical-accounts" class="table table-bordered table-striped">
```

Comment:
Can we squash this and the styling changes above into the previous commit please.